src/component.js

```diff
@@ -136,7 +136,7 @@ function renderComponent(component) {
 			newVNode,
 			oldVNode,
 			component._globalContext,
-			component._parentDom.ownerSVGElement !== undefined,
+			component._parentDom.ownerSVGElement !== undefined ? 2 : 1,
```

Comment:
Earlier MathML specs had a `.ownerMathElement` API that seems to have disappeared... Need to look into this more.

Reply:
Should exist according to https://www.w3.org/Math/DOM/mathml2/mathml-dom_ecma.html

Reply:
That's MathML 2, which was replaced with [MathML Core](https://www.w3.org/TR/mathml-core/)

Reply:
Testing `.namespaceURI` or `instanceof MathMLElement` is all we get at the moment.